linode/lketypes/framework_models.go

```diff
@@ -0,0 +1,121 @@
+package lketypes
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper/frameworkfilter"
+)
+
+type DataSourceModel struct {
+	ID           types.String `tfsdk:"id"`
+	Label        types.String `tfsdk:"label"`
+	Price        types.List   `tfsdk:"price"`
+	RegionPrices types.List   `tfsdk:"region_prices"`
+	Transfer     types.Int64  `tfsdk:"transfer"`
+}
+
+func (data *DataSourceModel) ParseLKEType(lkeType *linodego.LKEType,
```

Comment:
It looks like this function `ParseLKEType` is only used inside this package. Can we not expose it and make it `parseLKEType`? Same for `FlattenPrice`, `FlattenRegionPrices`.

Reply:
Good catch! Fixed this.